.github/workflows/black.yml

```diff
@@ -10,7 +10,7 @@ jobs:
     strategy:
       matrix:
         python-version:
-        - 3.6 # Match the version on the gateway node
+        - 3.6.7 # Match the version on the gateway node
```

Comment:
It looks like GitHub Actions doesn't know how to set up python 3.6.7 on ubuntu 22.04: https://github.com/equinix/terraform-metal-vsphere/actions/runs/4919295677/jobs/8786728960?pr=60#step:3:7